### ISSUE TITLE
Output folder should only contain the html files and related png images

### ISSUE THREAD

**Issue by cmaumet** [incf-nidash/nidmresults-fslhtml#7]
After running the viewer, the output folder contains the viewing files (i.e. HTML files, png images, etc.) as well as all the files that were part of the interogated NIDM-Results pack:
```
$ ls out/
ex_spm_default.nidm.zip

$ nidmviewerfsl out/ex_spm_default.nidm.zip out/view
['out/view/nidm.ttl']

$ ls -all out/view/
total 15280
drwxr-xr-x  19 cmaumet  staff      646  8 Dec 11:40 .
drwxr-xr-x   4 cmaumet  staff      136  8 Dec 11:40 ..
-rw-r--r--   1 cmaumet  staff     7574  8 Dec 11:40 ClusterLabels.nii.gz
-rw-r--r--   1 cmaumet  staff   856052  8 Dec 11:40 Contrast.nii.gz
-rw-r--r--   1 cmaumet  staff  1710388  8 Dec 11:40 ContrastStandardError.nii.gz
-rw-r--r--   1 cmaumet  staff     2666  8 Dec 11:40 DesignMatrix.csv
-rw-r--r--   1 cmaumet  staff      698  8 Dec 11:40 DesignMatrix.png
-rw-r--r--   1 cmaumet  staff    47378  8 Dec 11:40 ExcursionSet.nii.gz
-rw-r--r--   1 cmaumet  staff   790272  8 Dec 11:40 GrandMean.nii.gz
-rw-r--r--   1 cmaumet  staff    11607  8 Dec 11:40 Mask.nii.gz
-rw-r--r--   1 cmaumet  staff    15241  8 Dec 11:40 MaximumIntensityProjection.png
-rw-r--r--   1 cmaumet  staff  1594841  8 Dec 11:40 ReselsPerVoxel.nii.gz
-rw-r--r--   1 cmaumet  staff  1739878  8 Dec 11:40 ResidualMeanSquares.nii.gz
-rw-r--r--   1 cmaumet  staff    11607  8 Dec 11:40 SearchSpaceMask.nii.gz
-rw-r--r--   1 cmaumet  staff   854900  8 Dec 11:40 TStatistic.nii.gz
-rw-r--r--   1 cmaumet  staff      299  8 Dec 11:40 main.html
-rw-r--r--   1 cmaumet  staff   149048  8 Dec 11:40 nidm.ttl
-rw-r--r--   1 cmaumet  staff      602  8 Dec 11:40 postStats.html
-rw-r--r--   1 cmaumet  staff      627  8 Dec 11:40 stats.html
```

The files coming from the NIDM-Results pack that are not needed by the viewer should be deleted to save storage space.

**Comment by TomMaullin** [comment]
Hi @cmaumet , this will be the next issue I will address after the current PRs are merged. 

I thought I'd best confirm first though, should the output folder of the viewer only contain html files or should it be as close a replication of the gfeat directory given by fsl as possible (i.e. niftis and all)?

**Comment by cmaumet** [comment]
Hi @TomMaullin: only the rendered 'view', i.e. the html files (and possibly other files used by the html).

**Comment by TomMaullin** [comment]
@cmaumet Ah okay! Thank you for the clarification. Should images which have been embedded in the html also be included separately as png?

**Comment by cmaumet** [comment]
The goal is to have as few files as we can. If the images are embedded, no need to keep them again.

**Comment by TomMaullin** [comment]
Okay! I'll sort this this week hopefully!

**Comment by TomMaullin** [comment]
This was resolved in PR #26
